install_config/persistent_storage/persistent_storage_manila.adoc

```diff
@@ -0,0 +1,265 @@
+provisioning], and 
+link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
+
+[[installation-setup]]
+== Installation and Setup
+
+The feature is provided by an external provisioner. You must install and
+configure it in the {product-title} cluster.
+
+=== Starting the External Provisioner
+
+The external provisioner service is distributed as a container image and can be run in the
+{product-title} cluster as usual.
+
+To allow the containers managing the API objects, configure the required role-based access
+control (RBAC) rules as a cluster administrator:
+
+. Create a `ServiceAccount`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: manila-provisioner-runner
+----
+
+. Create a `ClusterRole`:
++
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: manila-provisioner-role
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+----
+
+. Bind the rules via `ClusterRoleBinding`:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manila-provisioner
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: manila-provisioner-role
+subjects:
+- kind: ServiceAccount
+  name: manila-provisioner-runner
+  namespace: default
+----
+
+. Create a new `StorageClass`:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "manila-share"
+provisioner: "externalstorage.k8s.io/manila"
+parameters:
+  type: "default" <1>
+  zones: "nova" <2>
+----
+<1> The link:https://docs.openstack.org/manila/latest/admin/shared-file-systems-share-types.html[Manila share type]
+the provisioner will create for the volume.
+<2> Set of Manila availability zones that the volume might be created in.
+
+Configure the provisioner to connect, authenticate, and authorize to the Manila service by using
+environment variables. Select the right combination of environment variables for your installation
+from the following list:
+[source]
+----
+OS_USERNAME
+OS_PASSWORD
+OS_AUTH_URL
+OS_DOMAIN_NAME
+OS_TENANT_NAME
+----
+
+[source]
+----
+OS_USERID
+OS_PASSWORD
+OS_AUTH_URL
+OS_TENANT_ID
+----
+
+[source]
+----
+OS_USERNAME
+OS_PASSWORD
+OS_AUTH_URL
+OS_DOMAIN_ID
+OS_TENANT_NAME
+----
+
+[source]
+----
+OS_USERNAME
+OS_PASSWORD
+OS_AUTH_URL
+OS_DOMAIN_ID
+OS_TENANT_ID
+----
+
+To pass the variables to the provisioner, use a `Secret`.
+Example of the `Secret` for the first variables combination:
```

Comment:
Suggest changing this to "The following example shows a `Secret` configured for the first variables combination":